roles/jd-server/src/main.rs

```diff
@@ -149,27 +107,44 @@ async fn main() {
         username,
         password,
     )));
-    let mempool_update_timeout = config.mempool_update_timeout;
+    let mempool_update_interval = config.mempool_update_interval;
     let mempool_cloned_ = mempool.clone();
     let (status_tx, status_rx) = unbounded();
     let sender = status::Sender::Downstream(status_tx.clone());
+    let empty_mempool_log_counter = 10 / mempool_update_interval.as_secs();
```

Comment:
How about (in pseudo-code):

```
let last_empty_mempool = now();
```

and then below 

```
if last_empty_mempool < 1.minute.ago() {
  log...
  last_empty_mempool = now()
}
```

Reply:
Do you mean to log the warning every minute, right?
If so, I can push this last changes

Reply:
Not every minute: no more than once a minute. If the interval is set to more than 1 minute it would log every interval.